lib/linguist/languages.yml

```diff
@@ -3736,6 +3736,15 @@ RenderScript:
   tm_scope: none
   ace_mode: text
   language_id: 323
+Ring:
+  type: programming
+  color: "#0e60e3"
+  extensions:
+  - .ring
+  - .rh
```

Comment:
@pchaigno Yes, the *.rh files are rarely used to define some constants instead of directly adding them to *.ring files.